src/utils.ts

```diff
@@ -242,6 +262,69 @@ export const parseTx = (tx: RawTx): FullTx => {
   return parsedTx;
 };
 
+/**
+ * Syncs the latest mempool
+ *
+ * @generator
+ * @yields {MempoolEvent}
+ */
+export async function* syncLatestMempool(): AsyncGenerator<MempoolEvent> {
+
+  logger.info(`Downloading mempool...`);
+  let success = true;
+  const mempoolResp = await downloadMempool();
+
+  if (!mempoolResp.success) {
+    yield {
+      success,
+      type: 'wait',
+      message: 'Could not download from mempool api',
+    };
+    return;
+  }
+
+  for (let i = 0; i < mempoolResp.transactions.length; i++) {
```

Comment:
Maybe we should keep track of the transactions we have already sent on a FIFO queue to prevent unnecessary lambda calls

I know that we currently have very few transactions on the `mempool` and even fewer of those are voided, but maybe the list will increase if someone is "voiding" transactions on purpose

Reply:
Maybe use `LRU` like `globalCache`?

Reply:
Resolved not to include a cache on this PR.